source/documentation/index.md

```diff
@@ -1,20 +1,10 @@
 # Reliability Engineering
 
-Reliability Engineering provide a shared platform to GDS teams comprising of tools to set up and maintain a service by:
+The Reliability Engineering team no longer exists and this documentation is now deprecated.
 
-* acquiring tools and where appropriate administers them like [Logit][]
-* running off-the-shelf services as internal SaaS such as [Prometheus][] and [Concourse][]
-* providing patterns and guidance like the [PaaS incident process][]
+Support for products that were previously offered by Reliability Engineering is now being done by a small number of kind volunteers. Please be kind and patient with them!
 
-To understand the context for our decisions and guidance refer to:
-
-* The [GDS Technology & Operations Principles][]
-* The [Reliability Engineering Strategy][]
-* The [GDS Technology & Operations Shared Responsibility Model][]
-* The [Service Manual][]
-* The [GDS Way][]
-
-The Reliability Engineering documentation found on this site is intended to help the rest of GDS find out what Reliability Engineering is and what we're doing. If you're a member of Reliability Engineering or just curious about our team processes and ongoing work then please take a look at our [Team Manual][].
+Some of this documentation may still be useful but you might instead find what you are looking for in:
```

Comment:
Should the following lines in the original version be included still? The text on line 7 above implies there would be following content but none is present.

Reply:
Yeah I think @idavidmcdonald assumed the below links were normal hyperlinks, but they're actually link references and don't display anything. I think I'd suggest just dropping this line entirely.

Reply:
Good spot, thanks. I have removed 3 of the internal links from this section in a new commit